spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -147,4 +161,47 @@
       expect(service_template.options[:config_info]).to include(saved_options)
     end
   end
+
+  describe '#update_catalog_item' do
+    it 'updates and returns the modified catalog item' do
+      service_template = prebuild_service_template
+      expect(described_class).to receive(:create_new_dialog)
```

Comment:
I am wondering if you don't return a new dialog here how can you assert the dialog name in line 178.